clusters/l3-sqnc/monitoring/release.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: kube-prometheus-stack
+spec:
+  interval: 1h
+  chart:
+    spec:
+      version: "57.x"
+      chart: kube-prometheus-stack
+      sourceRef:
+        kind: HelmRepository
+        name: prometheus-community
+      interval: 1h
+  install:
+    crds: Create
+  upgrade:
+    crds: CreateReplace
+  driftDetection:
+    mode: enabled
+    ignore:
+      # Ignore "validated" annotation which is not inserted during install
+      - paths: [ "/metadata/annotations/prometheus-operator-validated" ]
+        target:
+          kind: PrometheusRule
+  valuesFrom:
+  - kind: ConfigMap
+    name: flux-kube-state-metrics-config
+    valuesKey: kube-state-metrics-config.yaml
+  - kind: ConfigMap
+    name: monitoring-values
+    valuesKey: values.yaml
```

Comment:
We should probably create and set the default admin credentials for both prometheus and grafana dashboards in here using a secret encrypted for the l3-sqnc cluster.

Reply:
Done for grafana, but prometheus is very difficult to configure basic auth for through helm, especially with the `kube-prometheus-stack` chart (it would have to be implemented in nginx, not prometheus itself). Due to the limited value of exposing prometheus, I suggest we only expose grafana.

Reply:
Yep, accept this as a solution, prometheus dashboard is only really useful for debugging, so we can expose that via `kubectl port-forward`